src/app.py

```diff
@@ -25,29 +25,24 @@
 GOOGLE_CLIENT_SECRET = os.environ.get("GOOGLE_CLIENT_SECRET", None)
 GOOGLE_DISCOVERY_URL = os.getenv("GOOGLE_DISCOVERY_URL")
 LOGGING_LEVEL = logging._nameToLevel.get(os.getenv("LOGGING_LEVEL"), 'INFO')
-app.secret_key = os.getenv("APP_SECRET_KEY") or os.urandom(24)
+
+from .utils.authorization import secret_key, Generate_JWT, public_endpoint
```

Comment:
The secret key should REALLY be stored in the database - not on the backend server as an env var